testing/static-ip-conflict-with-pool.yaml

```diff
@@ -0,0 +1,80 @@
+# Environment file demonstrating how to pre-assign IPs to all node types
```

Comment:
How is this file used exactly? Is it intended for automated testing?

I suggest adding all sorts of errors to the file, like for instance non parsable IP addresses, IP ranges with lower bound higher than the upper bound, etc.